all_os_install_requirements.sh

```diff
@@ -0,0 +1,47 @@
+if ! which minikube 2>/dev/null ; then
+    curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+    chmod +x minikube
+    sudo mv minikube /usr/local/bin/.
+fi
+
+if ! which docker-machine-driver-kvm2 2>/dev/null ; then
+    curl -LO https://storage.googleapis.com/minikube/releases/latest/docker-machine-driver-kvm2
+    chmod +x docker-machine-driver-kvm2
+    sudo mv docker-machine-driver-kvm2 /usr/local/bin/.
+fi
+
+if ! which kubectl 2>/dev/null ; then
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+    chmod +x kubectl
+    sudo mv kubectl /usr/local/bin/.
+fi
+
+if ! which kustomize 2>/dev/null ; then
+    curl -Lo kustomize $(curl --silent -L https://github.com/kubernetes-sigs/kustomize/releases/latest 2>&1 | awk -F'"' '/linux_amd64/ { print "https://github.com"$2; exit }')
+    chmod +x kustomize
+    sudo mv kustomize /usr/local/bin/.
+fi
+
+# Download Ironic binary and Ironic inspector image
+
+mkdir -p "$IRONIC_DATA_DIR/html/images"
+pushd "$IRONIC_DATA_DIR/html/images"
+if [ ! -f ironic-python-agent.initramfs ]; then
+    curl --insecure --compressed -L https://images.rdoproject.org/master/rdo_trunk/current-tripleo-rdo/ironic-python-agent.tar | tar -xf -
+fi
+
+for IMAGE_VAR in IRONIC_IMAGE IRONIC_INSPECTOR_IMAGE ; do
+    IMAGE=${!IMAGE_VAR}
+    sudo podman pull "$IMAGE"
```

Comment:
you need to rebase, and be careful with the conflicts, podman should be replaced by the variable CONTAINER_RUNTIME